docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.10.2
```

Comment:
I just realized how different this build is from what Calypso runs. what if we added a new target in the `Makefile` for `make run-in-docker` and that spit out a nice console message saying, "Hey! This isn't out build environment so don't rely on this for tests!"